00-Install_and_Setup/requirements.txt

```diff
@@ -20,11 +19,11 @@ jupyterlab
 jupyter-offlinenotebook
 nbzip
 nbgitpuller
-astropy>=5.0
+astropy>=5.3
 asdf
 asdf-astropy
 gwcs>=0.16
 photutils>=1.3.0
 specutils>=1.1.1
-astroquery
+astroquery --pre
```

Comment:
Is there a known issue that we're avoiding with `--pre`?

Reply:
@bsipocz recommends to always install astroquery this way. They are on a continuous release model. I think the idea is that services are changing all the time, so they make releases quite infrequently and one should always get the pre-release version.